projects/dotcms-field-elements/src/components/dot-form/dot-form.e2e.ts

```diff
@@ -156,14 +179,33 @@ describe('dot-form', () => {
         });
     });
 
+    describe('rows', () => {
+        beforeEach(async () => {
+            element.setProperty('layout', fieldsMock);
+            element.setProperty('fieldsToShow', 'test');
+            await page.waitForChanges();
+        });
+
+        it('should have 2 rows', async () => {
+            const rows = await element.findAll('dot-form-row');
+            expect(rows.length).toBe(2);
+        });
+
+        it('should set values on dot-form-row', async () => {
+            const firstRow = (await element.findAll('dot-form-row'))[0];
```

Comment:
```suggestion
            const firstRow = (await element.find('dot-form-row'));
```

This will get you the first  row.